storage/txpool/src/service/interface.rs

```diff
@@ -27,15 +42,28 @@ pub enum TxpoolReadRequest {
 /// The transaction pool [`tower::Service`] read response type.
 #[expect(clippy::large_enum_variant)]
 pub enum TxpoolReadResponse {
-    /// Response to [`TxpoolReadRequest::TxBlob`].
-    ///
-    /// The inner value is the raw bytes of a transaction.
-    // TODO: use bytes::Bytes.
-    TxBlob(Vec<u8>),
+    /// A response containing the raw bytes of a transaction.
+    TxBlob { tx_blob: Vec<u8>, state_stem: bool },
 
-    /// Response to [`TxpoolReadRequest::TxVerificationData`].
+    /// A response of [`TransactionVerificationData`].
```

Comment:
Same here, also `TxBlob` should document what the corresponding request type is.

Also, `A response` -> `The response for [...]`